src/styles/common/_grid.scss

```diff
@@ -7,6 +7,11 @@
     stroke-dasharray: 4, 4;
 }
 
+.horizontal-grid-line--highlighted {
+    stroke: #ccc;
```

Comment:
Please, use one of the grey color variables we declare in https://github.com/Golodhros/britecharts/blob/master/src/styles/helpers/_variables.scss#L14 instead